app/views/components/_attachment_meta.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+  id ||= "attachment-meta-#{SecureRandom.hex(4)}"
```

Comment:
Do we use this for anything?

Reply:
Nope, not much. I thought would be good to have some consistency about the attributes that can be passed to a component.